admin-dev/themes/new-theme/package-lock.json

```diff
@@ -1242,13 +1236,13 @@
       "dev": true
     },
     "@types/bootstrap": {
-      "version": "5.0.12",
-      "resolved": "https://registry.npmjs.org/@types/bootstrap/-/bootstrap-5.0.12.tgz",
-      "integrity": "sha512-iowwPfp9Au6aoxS2hOgeRjXE25xdfLrTpmxzQSUs21z5qY3UZpmjSIWF4h8jPYPEXgZioIKLB2OSU8oWzzJAcQ==",
+      "version": "4.6.0",
```

Comment:
Is it normal the regression ?

Reply:
Yes, we're not using bootstrap 5, so types aren't adapted, we need types for the version 4, this was an error from the TS implementation (first PR)